.github/workflows/release.yml

```diff
@@ -4,6 +4,9 @@ on:
   release:
     types: [prereleased, released]
 
+env:
+  ORIGINAL_REPO_NAME: ${{ github.event.repository.full_name }}
```

Comment:
I think this env var should have the fixed string with the [upstream name ](https://github.com/newrelic/nri-kafka/blob/master/.github/workflows/push_pr.yml#L14)`nri-ecs` in this case, and IIRC this was used because when open a PR from a fork that `${{ github.event.repository.full_name }}` changed , or it was because some tests done in forked repositories. 
IMO this env var doesn't make sense at all but I think is fine to add it for consistency.

Reply:
I would use directly `github.event.repository.full_name` 
In the 3 repos, I saw that such env is the most common one, but it is quite confusing that in some we have even two!

```
  ORIGINAL_REPO_NAME: ${{ github.event.repository.full_name }}
  REPO_FULL_NAME: ${{ github.event.repository.full_name }}
```

Reply:
totally confusing and IMO is not needed, this is why I suspect that they are leftovers from testing the pipeline from a fork

Reply:
I'm fine with both approaches, remove them at all and use native gha vars, or keep the for consistency using the literal string

Reply:
decided offline to use the literal string for consistency